processing/src/main/java/io/druid/query/filter/LongValueMatcherColumnSelectorStrategy.java

```diff
@@ -32,7 +31,20 @@ public ValueMatcher makeValueMatcher(final BaseLongColumnValueSelector selector,
   {
     final Long matchVal = DimensionHandlerUtils.convertObjectToLong(value);
     if (matchVal == null) {
-      return BooleanValueMatcher.of(false);
+      return new ValueMatcher()
```

Comment:
Identical code appears in all three variants (Long, Double and Float), could be extracted as an utility ValueMatcher